.github/workflows/deploy.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           context: .
           push: true
           tags: |
-            ${{ steps.vars.outputs.docker_image }}:${{ steps.vars.outputs.sha_short }}
+            ${{ steps.vars.outputs.docker_image }}:1.${{ github.run_number }}.${{ steps.vars.outputs.sha_short }}
```

Comment:
I used a different approach here: https://github.com/netwerk-digitaal-erfgoed/network-of-terms-demo/commit/6ad2e4202fac4d27aa77bca413542ca5f36937bc

Let’s standardise on a single solution, which will be easier for both dev and ops. I prefer the `--date=format:...` because it doesn’t rely on the build system. Who knows we’ll move away from GitHub some time, which means will lose the `github.run_number` increment. 

What are your thoughts?